R/app_ui.R

```diff
@@ -10,7 +10,8 @@ app_ui <- function(request) {
     golem_add_external_resources(),
     # Your application UI logic
     fluidPage(
-      golem::golem_welcome_page() # Remove this line to start building your UI
+      h1("Hello!"),
```

Comment:
Test comment make this prettier
```suggestion
      h1("Hello!  YAY!"),
```